### PR TITLE
Add footer in embed mode

### PR DESCRIPTION
**Pull request by caseycesari**
## Overview

Display the footer, which has a different design and contents to the main footer, when the map is in embed mode.

Connects #1336 

## Demo

image

## Testing Instructions

- View the app in embed mode, and verify the embed footer is displayed. 
- Ensure the app 

## Checklist

- [x] `fixup!` commits have been squashed
- [x] CI passes after rebase
- [x] CHANGELOG.md updated with summary of features or fixes, following [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) guidelines

### CODE REVIEW

File: src/app/src/components/EmbeddedFooter.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    <footer className="footerContainerEmbedded results-height-subtract" xs={12}>
+        <img
+            className="footer-image"
+            src={logo}
+            alt="Powered by Open Apparel Registry"
+        />
+    </footer>
+);
+
+export default EmbeddedFooter;
```
Comment (caseycesari) [inline src/app/src/components/EmbeddedFooter.jsx:14]: I thought about integrating this into the existing footer component, but the code for that component is somewhat complex, and it seemed simpler to make a new component.

File: src/app/src/App.jsx [hunk 2]
```diff
@@ -94,7 +95,7 @@ class App extends Component {
         const { fetchingFeatureFlags, embed } = this.props;
 
         const mainPanelStyle = embed
-            ? { ...appStyles.mainPanelStyle, bottom: 0, top: 0 }
+            ? { ...appStyles.mainPanelStyle, top: 0 }
```
Comment (caseycesari) [inline src/app/src/App.jsx:98]: If the `bottom` rule is overridden in embed mode, the footer overlaps with the app.

File: src/app/src/components/EmbeddedFooter.jsx (hunk 1 above)
Comment (TaiWilkin) [reply src/app/src/components/EmbeddedFooter.jsx:14]: That seems like a good call

**Review by TaiWilkin — APPROVED** [review]
Working well!

File: src/app/src/components/EmbeddedFooter.jsx (hunk 1 above)
Comment (TaiWilkin) [reply src/app/src/components/EmbeddedFooter.jsx:14]: That seems clearer in terms of readibility, as well

**Pull request merged**